tests/aderyn.config.json

```diff
@@ -0,0 +1,4 @@
+{
+    "use_detectors": "push-zero-opcode,useless-public-function",
+    "scope_file": "../tests/scope.txt"
```

Comment:
I'm toying with the idea of defining the scope in this config file instead of a separate one. What do you think @TilakMaddy ?

Reply:
we can have both : )

scope_file ... (like it is right now)
scope .. (like you mentioned)

Reply:
also if it's a scope file it can be anything - people would just have to drop in this config. So it's more like we adjust for them rather than they adjust for us : )

Reply:
let's have a separate PR for that : ) This is more for like registry and introducing aderyn.config.json : )